crm_rma_prodlot_invoice/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+
+=========================
+CRM Claim Prodlot Invoice
+=========================
+
+When an invoice is created from sale order, picking and procurement information
+is kept into invoice line.
```

Comment:
@osvalr change this please, remember that hte structure was changed and now the invoice line id is available in stock.production.lot model.